build/repo.props

```diff
@@ -7,6 +7,7 @@
     <DisableDefaultTargets>true</DisableDefaultTargets>
     <DisableDefaultItems>true</DisableDefaultItems>
     <BuildSolutions>false</BuildSolutions>
+    <BuildSiteExtensions Condition="'$(SharedFxRid)' == 'win-x64' OR '$(SharedFxRid)' == 'win-x86'">true</BuildSiteExtensions>
```

Comment:
SharedFxRid is defined in `src\Framework\src\Microsoft.AspNetCore.App.shfxproj` you might want to move it here.

Reply:
It's also defined in eng\targets\RuntimeIdentifiers.props, which is imported at the top of this file. This should be okay.